src/deepforest/main.py

```diff
@@ -959,15 +872,15 @@ def on_validation_epoch_end(self):
 
         if self.current_epoch % self.config.validation.val_accuracy_interval == 0:
             if len(self.predictions) > 0:
-                self.predictions = pd.concat(self.predictions)
+                predictions = pd.concat(self.predictions)
```

Comment:
Callback order isn't guaranteed, best practice is to not mutate the state of the module (i.e. we can't rely on this callback concatenating the predictions).

Reply:
that seems fair